modules/rosa-getting-started-revoke-user-access.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-revoke-user-access_{context}"]
+= Revoking user access to a cluster
+
+You can revoke cluster access for an identity provider user by removing them from your configured identity provider.
+
+You can configure different types of identity providers for your ROSA cluster. The following example procedure revokes cluster access for a member of a GitHub organization that is configured for identity provision to the cluster.
```

Comment:
Is this first sentence necessary? It was when you were adding access, but if you're removing access, you've already created the applicable identity provider type.

Reply:
My view is that the reader might not have read the section on creating an identity provider. They might have been asked to remove it after someone else created it. I also think that it's useful to keep here because it highlights that the procedure uses only one of a number of supported types in order to provide an example. Based on this, I will keep the line in the section.